ONSdigital/response-operations-ui
# Add js testing framework

# Motivation and Context
Added a Javascript testing framework (Jest) to allow us to start covering Javascript used on the frontend (currently not covered).

# What has changed
* Installed Jest
* Added Jest Gulp task
* Added test to NPM
* Added test to Gulp 'test' and 'watch' functions
* Wrote example test file to help explain tests to users

# How to test?
* Remove `.skip` as per comment in example test file.
* Run jest tests to confirm they run (run `npm test`, `npm run gulp test`)
* No acceptance tests involved

# Links
https://trello.com/c/8Mc2jeAx
